src/main/java/org/apache/sling/jcr/packageinit/impl/ExecutionPlanRepoInitializer.java

```diff
@@ -163,6 +164,14 @@ public void processRepository(SlingRepository slingRepository) throws Exception
                         builder.with(session);
                         ExecutionPlan xplan = builder.execute();
                         if (xplan.getTasks().size() > 0) {
+                            if (xplan.hasErrors()) {
+                                for (PackageTask task : xplan.getTasks()) {
+                                    if (PackageTask.State.ERROR.equals(task.getState())){
+                                        logger.error("Error during installation of {}: {}", task.getPackageId().toString(), task.getError().toString());
```

Comment:
Please rather add those as suppressed exceptions via https://docs.oracle.com/javase/7/docs/api/java/lang/Throwable.html#addSuppressed(java.lang.Throwable) to the ISE. They already get logged within https://github.com/apache/jackrabbit-filevault/blob/beb4a72adcf963c1da6ed9604c18b8b5b69ee57c/vault-core/src/main/java/org/apache/jackrabbit/vault/packaging/registry/impl/PackageTaskImpl.java#L113.

Reply:
@kwin  ah yes makes sense.. just pushed a corresponding update